packages/ts/form/src/Validators.ts

```diff
@@ -310,7 +310,7 @@ export class Digits<T> extends AbstractValidator<T> {
 
   override validate(value: T): boolean {
     return (
-      String(toFloat(String(value))).replace(/(.*)\.\d+/u, '$1').length <= this.integer &&
+      String(toFloat(String(value))).indexOf('.') <= this.integer &&
```

Comment:
This change seems OK to me, anyway I question both implementation about negative number, as the minus sign seems to be taken into account here. 

There are no tests for negative numbers indeed.

I'm going to open an issue for that.

Reply:
I believe that we shouldn't do such kind of changes in this PR. Could we ignore the Sonar quality gates and create an issue for these issues?

Reply:
Except that it's blocking merging

Reply:
Concerning the minus sign yes, I opened a different issue.

Reply:
Hmm, I was sure we could ignore Sonar if it's better for us 🤔

Reply:
@platosha what do you think?

Reply:
let's address in a separate PR

Reply:
Sonar is red but not blocking I think

Reply:
OK so I'm going to revert my changes.